frontend/appflowy_tauri/src/appflowy_app/components/layout/NavigationPanel.tsx

```diff
@@ -38,48 +68,119 @@ export const NavigationPanel = () => {
 
         <div className={'px-2 flex flex-col'}>
           {folders.map((folder, index) => (
-            <div key={index}>
-              <div className={'px-4 py-2 my-2 flex items-center justify-between rounded-lg hover:bg-surface-2'}>
-                <div
-                  onClick={() => setFolderOpen(folder.id, !isFolderOpen[folder.id])}
-                  className={'flex items-center flex-1 min-w-0 cursor-pointer '}
-                >
+            <div key={index} className={'my-2'}>
+              <div
+                onClick={() => renamingFolderId !== folder.id && setFolderOpen(folder.id, !isFolderOpen[folder.id])}
```

Comment:
I think this would be easy to understand if extracting this to a single function

Reply:
refactored